scout/constants/gene_tags.py

```diff
@@ -15,6 +15,20 @@
     ("other", "other - Other"),
 )
 
+# Associates an inheritance model with a color, for displaying inheritance on Variant page
+INHERITANCE_PALETTE = {
+    "AD": {"bgcolor": "bg-red", "text_color": "text-white"},
+    "AR": {"bgcolor": "bg-blue", "text_color": "text-white"},
```

Comment:
Ah, and for the models, one would want `AR_hom` and `AD_dn` to really stand out. Here, for the litterature inheritance pattern for the gene, it does not matter so much, as long as it is easy to compare with what the individual actually has.

Reply:
We can still keep them blue and red. But the bootstrap colors are basically these, so if we want to color by categories ( a gradient) then we need to define these colors in the css. Let's see!